apiary.apib

```diff
@@ -1556,41 +3369,1468 @@ Fetches a bundle metadata by identifier
+                    "sent": true,
+                    "validUntil": 1633594494,
+                    "createdBy": "0x0C6C7264E509430bcFa2469bA4eB738c94e6C75A",
+                    "createdOn": 1633594494
+                }
+            ],
+            "meta": {
+                "code": 200,
+                "count": 1
+            },
+            "pagination": {
+                "hasNext": false,
+                "hasPrevious": false
+            }
+        }
+
+
+## /organization/invite/{inviteId}/exists
+#### Check if invite exists [GET]
++ Parameters
+
+    + inviteId (number, required) - the invitaion Id
++ Response 200 (application/json)
+
+    + Attributes
+
+        + data (boolean) - shows if invite exists
+        + meta (object) - the metadata
+            + code (string) - the status code
+
+    + Body
+
+            {
+                "data": true,
+                "meta": {
+                    "code": 200
+                }
+            }
+
+## /organization/invite/{inviteId}/accept
+#### Accept the invitation [POST]
+* Parameters
+
+    + inviteId (number, required) - the invitation Id
+
++ Request
+
+    + Attributes
+
+        + address (string) - the account of the one who accepts invitation
+
+    + Body
+
+            {
+                "address": "0x203C394e59915f85B9c222f6E15ce1FFF699559E"
+            }
+
++ Response 200 (application/json)
+
+    Accepted.
+
+    + Body
+
++ Response 404 (application/json)
+
+    Invite doesn't exist or the metadata for it is missing
+
+    + Body
+
+## /organization/invite/{inviteId}
+#### Delete the invitation [DELETE]
++ Parameters
+
+    + inviteId (number, required) - the invitation Id
+
++ Response 200 (application/json)
+
+    Deleted.
+
+    + Body
+
++ Response 404 (application/json)
+
+    Invite doesn't exist or the metadata for it is missing
+
+    + Body
+
+## /organization/invite/resend
+__Warning__ super_account permission is required
+#### Resend all invitation [GET]
++ Response 200 (application/json)
+
+    + Body
+
+
+## /organization/invite/
+__Warning__ super_account permission is required
+#### Create invitation Buld [POST]
++ Request
+
+    + Attributes
+
+        + email (string) - the email on which invite must be sent
+
+    + Body
+
+            {
+                "email": "test+testOrgRequest1@ambrosus.io",
+            }
+
++ Response 201 (application/json)
+
+    Created.
+
+    + Body
+
+## Group Organization Request
+Request is the request to join organization. Consists of:
+* address - string, required - the account address that requests to join.
+* title - string, required - the title of organization that is requested
+* email - string, required - the email of requet sender
+* message - string, required - the message to organization
+* createdOn - number, required - the unix timestamp of time when request was created
+* refused - boolean, required - the status of request
+
+## /organization/request
```

Comment:
Same as /organization/request/ ?

Reply:
There is GET and POST calls description. I'll try to make it unambiguous